app/lib/core/view_models/welcome_view_model.dart

```diff
@@ -1,3 +1,18 @@
+import 'package:app/core/services/navigation/nav_service.dart';
+import 'package:app/core/services/service_locator.dart';
 import 'package:app/core/view_models/base_view_model.dart';
+import 'package:app/ui/views/signin/sign_in.dart';
+import 'package:app/ui/views/signup/signup.dart';
 
-class WelcomeViewModel extends BaseViewModel {}
+// TODO: To Mansi - Add something if needed
```

Comment:
- [ ] Add a task number for this. What is to be added?

Reply:
This is not defined task right now, but @mansiPaTeLL is creating a logo that will be put in welcomeview screen. To do that, I assume we might need some kind of functionality inside view model

Reply:
OK. I want to be more picky about how many TODOs we have so we don't accidentally miss something.